src/OrbitDB.js

```diff
@@ -45,16 +45,23 @@ class OrbitDB {
       ? new options.broker(this._ipfs) // eslint-disable-line
       : new Pubsub(this._ipfs, this.id)
     this.directory = options.directory || './orbitdb'
-    this.keystore = options.keystore
-    this.caches = { 'default': options.cache }
     this.storage = options.storage
-    this.stores = {}
     this._directConnections = {}
+
+    this.caches = { 'default': options.cache }
+    // flag so that the store class doesn't close it
+    if(this.caches['default']) this.caches['default'].isDefault = true
```

Comment:
Is `isDefault` used anywhere?

Reply:
Yes, in `orbit-db-store` here: https://github.com/orbitdb/orbit-db-store/blob/master/src/Store.js#L193

Basically this prevents `db.drop` and `db.close` from inadvertently closing the "master" or "default" store managed by OrbitDB. This is in line with the overall refactor of not creating a separate cache store for every DB _unless_ `options.directory` is passed into `OrbitDB.create` or `OrbitDB.open`, which is another performance optimization